ui/packages/catalog-ui-search/src/main/webapp/react-component/container/searches-container/search-root.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Provider } from 'react-redux'
+import store from './store'
+import SearchesContainer from './searches-container'
+
+export default function SearchRoot() {
+  return (
+    <Provider store={store}>
+      <SearchesContainer />
+    </Provider>
+  )
+}
```

Comment:
✏️ Not that this makes a difference as it currently is, but this could easily be in TypeScript.